### ISSUE TITLE
A11y: Flashed messages are not screen reader friendly

### ISSUE THREAD

**Issue by lhenze** [inveniosoftware/invenio-app-rdm#1162]
Package version: <= 7

## Describe the bug
"When creating an account or verifying an email address, alerts that appear at the top of the page are not announced to the screen reader user. They will need to navigate through the page until they come upon the information." (From NYU A11y review)

## Steps to Reproduce
1. Click "Sign Up" button in top navbar; 
2.  Enter an email and password
3.  Click "Sign Up" submit button
4. Or many other interactions throughout the application

## Expected behavior

Markup is coded in such a way that the screen reader recognizes the "flashed message" as an alert
## Screenshots (if applicable)
![Screen Shot 2022-01-14 at 5 16 57 PM](https://user-images.githubusercontent.com/86330/149592746-3e287fb4-35c6-4055-94a2-c97ddd8a75be.png)

## Related WCAG Criteria

3.2.6 Status Changes (WCAG 2.1 numbers this as [4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)): In content implemented using markup languages, status messages can be programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus.

## Additional context

The solution may be adding role="alert" to the "flashed" messages, for example:
<div class="ui info flashed message" role="alert">
